app/templates/components/events/view/overview/event-setup-checklist.hbs

```diff
@@ -0,0 +1,56 @@
+<div class="content">
+  <div class="header">{{t 'EVENT SETUP CHECKLIST'}}</div>
+</div>
+<div class="ui segments content">
+  <div class="ui teal inverted segment">
+    <i class="large checkmark middle aligned icon"></i>
+    <div class="content">
+      <div class="right floated content">
+        <div class="ui button"><i class="edit icon"></i>Edit</div>
+      </div>
+      <b class="header">{{t 'Basic Details'}}</b>
+      <div class="description">{{t 'Basic information such as title,date,venue,etc.'}}</div>
+    </div>
+  </div>
+  <div class="ui grey inverted segment">
+    <i class="large checkmark middle aligned icon"></i>
+    <div class="content">
```

Comment:
Use grid UI for checklist item,use rows & columns. https://semantic-ui.com/collections/grid.html